.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   release_carthage:
     # Specify the Xcode version to use
     macos:
-      xcode: "10.0.0"
+      xcode: "9.2.0"
```

Comment:
How does changing the xcode version help?

Reply:
Xcode 9.3 [isn't packaged with `docsetutil`](https://github.com/tomaz/appledoc/issues/628), which we use for generating appledoc. We decided to force-downgrade the Xcode version we're building with.

Reply:
@palpatim We already made the change for that for our docs workflow, but this one would also use the older xcode for our Carthage builds.